pyproject.toml

```diff
@@ -246,13 +246,15 @@ gud,
 hax,
 hist,
 hve,
+inpt,
```

Comment:
Used in Stix notebook.  Didn't make the change to `input` because `input` is a Python built-in.